ignite/contrib/handlers/tensorboard_logger.py

```diff
@@ -527,3 +527,46 @@ def __call__(self, engine: Engine, logger: TensorboardLogger, event_name: Union[
             logger.writer.add_histogram(
                 tag=f"{tag_prefix}grads/{name}", values=p.grad.detach().cpu().numpy(), global_step=global_step
             )
+
+
+class TrainerStateHandler(BaseOutputHandler):
+    """Helper handler to log state attributes of the trainer.
```

Comment:
I think instead of this new class we can insert additional argument `state_attributes` to `OutputHandler`.

Reply:
fixed!